pkg/autoid_service/autoid.go

```diff
@@ -185,6 +201,12 @@ func (alloc *autoIDValue) rebase4Unsigned(ctx context.Context,
 	}
 	// Satisfied by alloc.end, need to update alloc.base.
 	if requiredBase <= uint64(alloc.end) {
+		logutil.BgLogger().Info("rebase4Unsigned in memory",
```

Comment:
If there are a large number of explicit inserts here, will there be too many logs here?

Reply:
+1